README.md

```diff
@@ -129,21 +129,140 @@ Finally, run the replicator:
 poetry run cdf_fabric_replicator config.yaml
 ```
 
-# Docker
+## Visual Studio Code
+
+Alternatively, if you are using Visual Studio Code, just open the folder of the root directory of your project.
+
+You must still install Poetry as mentioned above in addition to the [Python Extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) for VSCode.
+
+The included ".vscode/launch.json" file will add "poetry install" and "poetry run" to your "Run and Debug" tab. They can be used in place of the manual command line statements above, and will attach the VSCode debugger.
+
+For more information see [Debugging in Visual Studio Code](https://code.visualstudio.com/Docs/editor/debugging).
+
+# Building and Deploying with Docker on AKS
+
+## Arm Architecture
 
 If you are running the Docker Container on ARM Architecture, you will need to use platform emulation for the Docker run command. You can do this by running the following command:
 
 ```
 docker run -i -t <image-name> --platform linux/arm64 
 ```
-=======
-## Visual Studio Code
 
-Alternatively, if you are using Visual Studio Code, just open the folder of the root directory of your project.
+## Creating an AKS(Azure Kubernetes Service) Cluster with Managed Identity and Attached ACR(Azure Container Registry)
 
-You must still install Poetry as mentioned above in addition to the [Python Extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) for VSCode.
+Azure Kubernetes Service (AKS) can use Azure Managed Identities to interact with other Azure services. This eliminates the need to manage service principals and rotate credentials. The Fabric Replicator requires managed identity on AKS to be enabled to run.
 
-The included ".vscode/launch.json" file will add "poetry install" and "poetry run" to your "Run and Debug" tab. They can be used in place of the manual command line statements above, and will attach the VSCode debugger.
+To create an AKS cluster with Managed Identity and an attached ACR, you can use the Azure CLI:
 
-For more information see [Debugging in Visual Studio Code](https://code.visualstudio.com/Docs/editor/debugging).
+```bash
+az aks create -g MyResourceGroup -n MyManagedCluster --generate-ssh-keys --attach-acr MyACR --enable-managed-identity
```

Comment:
Should we also enable user-assigned managed identities in addition to the system-assigned identities?

Reply:
We were only using system assigned right now and for the time being we could keep that option closed off until we use them